api/server/controllers/AuthController.js

```diff
@@ -3,25 +3,25 @@ const userService = require('../services/UserService');
 const AuthService = require('../services/authService');
 const util = new Util();
 
-
 const { regValidation, loginValidation } = require('../validators/validate');
 
 class AuthController {
-
   static async signUp(req, res) {
-    // Joi checks User Input
-    // const validateInput = regValidation.validate(req.body);
-    // if (validateInput.error) {
-    //   throw (validateInput.error.message, 401);
-    // }
-    console.log("iiiiiiiiiiiiiiiiiiiiii")
-
-
-    const result = await AuthService.signup(req.body);
-    res.status(201).send(response("User created", result));
+    console.log('iiiiiiiiiiiiiiiiiiiiii');
+    try {
+      // Joi checks User Input
+      const validateInput = regValidation.validate(req.body);
+      if (validateInput.error) {
+        // return util.setError (validateInput.error.message, 401);
+        util.setError(400, validateInput.error.message);
+        return util.send(res);
+      }
+
+      const result = await AuthService.signup(req.body);
+      res.status(201).send(response('User created', result));
+    } catch (error) {
+      return util.setError(409, 'Something went wrong');
```

Comment:
The catch block is missing  `return util.send(res)`;
An omission on my part. Please include locally.